### PR TITLE
Better German translation

### PR DESCRIPTION
**Pull request by ThexXTURBOXx**
Fixes some small errors

### CODE REVIEW

File: src/main/resources/translations/german.json [hunk 1]
```diff
@@ -111,17 +111,17 @@
   "NEW_JAVA_PLUGIN": "Neues Java-Plugin",
   "SAVE_AS": "Speichern als...",
   "SAVE": "Speichern...",
-  "HIGH_CONTRAST_LIGHT": "Licht mit hohem Kontrast",
-  "DARK_ALT": "Dunkel-Alt",
-  "ONE_DARK_THEME": "Ein dunkles Thema",
+  "HIGH_CONTRAST_LIGHT": "Hell mit hohem Kontrast",
+  "DARK_ALT": "Dunkel Alternativ",
+  "ONE_DARK_THEME": "One Dunkles Thema",
```
Comment (Janmm14) [inline src/main/resources/translations/german.json:116]:
```suggestion
  "ONE_DARK_THEME": "Ein dunkles Thema",
```

**Comment by Janmm14** [comment]
"One Dunkel" is not appropiate. either use complete german translation or original english name One Dark

**Comment by ThexXTURBOXx** [comment]
Better

**Comment by Konloch** [comment]
thank you!

**Pull request merged**